I am running into issue reproducing the build failure locally. In the travis build it selects `gcc4.8` on https://travis-ci.com/andrewhop/s2n/jobs/150705795#L3181 but even after installing 4.8 and telling clang to use it with --gcc-toolchain=/usr/lib/gcc/x86_64-linux-gnu/4.8 clang fails with `Fuzzer/FuzzerDefs.h:15:10: fatal error: 'cassert' file not found`

--- a/tests/fuzz/LD_PRELOAD/s2n_server_fuzz_test_overrides.c
+++ b/tests/fuzz/LD_PRELOAD/s2n_server_fuzz_test_overrides.c
@@ -47,13 +47,13 @@ int s2n_constant_time_equals(const uint8_t *a, const uint8_t *b, uint32_t len)
     return 1;
 }
 
-int s2n_rsa_client_key_recv(struct s2n_connection *conn)
+int s2n_rsa_client_key_recv(struct s2n_connection *conn, struct s2n_blob *shared_key)
 {
     /* Perform the original function */
-    typedef int (*orig_s2n_rsa_client_key_recv_func_type)(struct s2n_connection *conn);
+    typedef int (*orig_s2n_rsa_client_key_recv_func_type)(struct s2n_connection *conn, struct s2n_blob *shared_key);
     orig_s2n_rsa_client_key_recv_func_type orig_s2n_rsa_client_key_recv;
     orig_s2n_rsa_client_key_recv = (orig_s2n_rsa_client_key_recv_func_type) dlsym(RTLD_NEXT, "s2n_rsa_client_key_recv");
-    int original_return_code = orig_s2n_rsa_client_key_recv(conn);
+    int original_return_code = orig_s2n_rsa_client_key_recv(conn, shared_key);
 
     /* Then, overwrite the RSA Failed flag to false before returning, this will help fuzzing code coverage. */
     conn->handshake.rsa_failed = 0;

